src/index.html

```diff
@@ -6,7 +7,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-sun calendar--month-length-29">
```

Comment:
Check correct name for the start day modifier according to the task requirements